atomic_scanners/container-capabilities-scanner/scanner.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# import docker
+import json
+import logging
+import os
+import subprocess
+import sys
+
+from Atomic import run
+from datetime import datetime
+
+INDIR = "/scanin"
+OUTDIR = "/scanout"
+IMAGE_NAME = os.environ.get("IMAGE_NAME")
+
+# object of atomic run
+run_object = run.Run()
+run_object.image = IMAGE_NAME
+
+# client = docker.Client(base_url="unix:///var/run/docker.sock")
+
+# set up logging
+logger = logging.getLogger("container-pipeline")
+logger.setLevel(logging.DEBUG)
+
+ch = logging.StreamHandler(sys.stdout)
+ch.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+
+# image UUID
+UUID = run_object.get_input_id(IMAGE_NAME)
+
+
+class RunLabelException(Exception):
+    """
+    This is a custom exception which should be raised when RUN label for
+    container image under scan is empty
+    """
+
+    def __init__(self, message):
+        super(RunLabelException, self).__init__(message)
+
+
+def check_image_for_run_label(image):
+    """
+    If image under scan doesn't have a RUN label, raise RunLabelException
+    """
+    run_label = run_object.get_label("RUN")
+
+    if run_label == "":
+        raise RunLabelException("{} doesn't have a RUN label".format(image))
+    return run_label
```

Comment:
Container image is not really run. Only it's `RUN` label is inspected. We don't spin up a new container from it.